src/main/webapp/app/exam/participate/summary/exercises/quiz-exam-summary/quiz-exam-summary.component.ts

```diff
@@ -45,6 +51,7 @@ export class QuizExamSummaryComponent implements OnInit {
                 this.exerciseWithSolution = response.body!;
             });
         }
+        this.showMissingResultsNotice = this.exam!.publishResultsDate!.isBefore(moment()) && !this.exercise.studentParticipations[0].results[0];
```

Comment:
`publishResultsDate` can be null/undefined which would throw an error here. If it is undefined / null, we do not show results. You also might want to use serverDate.now() instead of moment()